### DISCUSSION TITLE
购买专业版后一直是免费版，麻烦帮处理下！！！

### DISCUSSION THREAD

**Discussion by liangzi16** [getlantern/lantern-forum-cn#982] [General]
账号：39053595@qq.com ,7月购买一年专业版，使用一个多月后再使用就一直是免费版。付款记录见附件
![微信图片_20240923095438](https://github.com/user-attachments/assets/a227473f-1d6a-4c1c-b8af-ff0aedaa45d6)
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@liangzi16 您好. 好的. 我已经帮您提交问题上去了. 等待处理. 非常抱歉啊. 我们会尽快给您解决问题的.
